CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## 1.1.0 (unreleased)
 
+
+### Added
+- Add support for the general sibling combinator
+  ([#302](https://github.com/jjriv/emogrifier/pull/302))
```

Comment:
Could you please move this new entry as first entry of the existing "Added" section?